lib/ContainerManager.js

```diff
@@ -200,15 +207,19 @@ Server.prototype.runBuild = function* (build, log) {
     image: image,
     imageConfig: self.config.images[image],
 
-    // global config
+    // Global configuration for what binds to expose to all containers.
     binds: self.config.binds,
     // attachLogs: true, // set to true to attach 'docker -f logs' to stdout of this process
 
-    assetsUrl: this.config.assets.url
+    // TODO: Can we have a more generic way of passing plugin config through
+    // without explicit knowledge of plugin configuration here?
+    assetsUrl: this.config.assets.url,
```

Comment:
Actually, I meant independent of any individual plugin - a way to make our plugins less special - so that if someone wanted to add an "environment setup" step (non-user/repo specified step) that would be possible purely through config.

I think it's out of scope here, though I'll leave the TODO for when we get to the refactor around the event emitting/kafka stuff.